examples/client/client.c

```diff
@@ -3075,13 +3078,14 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
     if (sendGET) {
         printf("SSL connect ok, sending GET...\n");
-        msgSz = sizeof("GET /index.html HTTP/1.0\r\n\r\n");
-        XSTRNCPY(msg, "GET /index.html HTTP/1.0\r\n\r\n", msgSz);
-        msg[msgSz] = '\0';
+        char msgGet[GETMSGSZ] = "GET /index.html HTTP/1.0\r\n\r\n";
```

Comment:
I'd suggest removing the `GETMSGSZ` macro and rely on the more common `sizeof(msgGet)-1` pattern instead if you need the string length.

While `strncpy(..., GETMSGSZ-1)` probably works because it terminates early at the `\0` byte, it looks a bit odd.

Reply:
Done, thank you for your feedback!